tools/SetupFlow/DevHome.SetupFlow/Services/DevDriveManager.cs

```diff
@@ -274,7 +275,7 @@ private DevDrive GetDevDriveWithDefaultInfo()
                 TelemetryFactory.Get<ITelemetry>().Log(
                                               "DevDrive_Insufficient_DiskSpace",
                                               LogLevel.Critical,
-                                              new EmptyEvent());
+                                              new DevDriveInsufficientDiskSpaceEvent());

```

Comment:
I believe `EmptyEvent` is used in other places.  Do those need to be changed?

Reply:
I checked EmptyEvent, and it was used only by LogCritical, which is used by 5 events in SetupFlow. Two of those events have correct tagging and are not bugs. The other three I suspect may be bugs or have not yet generated a bug. 

I like Vineeth/Amir's recommendation of adding a privacy tag parameter. Alternately we enforce event classes for every event by removing EmptyEvent and requiring LogCritical have an event class passed into it.

In either case, I will address that if/when those three events become a bug.